test/games/prime1/exporter/test_prime_game_exporter.py

```diff
@@ -15,7 +16,8 @@ def test_patch_game(mocker, tmp_path, use_echoes_models):
     mock_patch_iso_raw: MagicMock = mocker.patch("py_randomprime.patch_iso_raw")
```

Comment:
You should also mock your patcher module and assert that it is being called with the values you expect. You can use this line here as a reference.

Ideally, you'd also add another parametrize to have a test with the enemy attributes being enabled or disabled. Should be similar to how `use_echoes_models` is done.